src/olympia/editors/helpers.py

```diff
@@ -718,6 +718,7 @@ def confirm_auto_approved(self):
         is incrementing AddonApprovalsCounter, which also resets the last
         human review date to now, and log it so that it's displayed later
         in the review page."""
+        self.version = self.addon.current_version
```

Comment:
I don't feel confident having random side-effects in classes, right now only `reject_multiple_versions` set's `self.version` and usually we have a `set_` helper for that, although they explicitly modify data in the database… hmm - *thinking loud* - but then again the `ReviewHelper` is initialized for every new form / page instance so that should not be a problem I think. Nevermind if that's the case :)